packages/docusaurus-theme-classic/src/theme/BlogTagsPostsPage/index.js

```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import Layout from '@theme/Layout';
 import BlogPostItem from '@theme/BlogPostItem';
 import Link from '@docusaurus/Link';
+import {pluralize} from '@docusaurus/utils'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
This bites me again, I need to somehow solve this ESLint error, but I still don’t get how. Any ideas for this? @yangshun 

```
error  '@docusaurus/utils' should be listed in the project's dependencies. Run 'npm i -S @docusaurus/utils' to add it  import/no-extraneous-dependencies
```

Reply:
Aha, I found a solution! I’ll make a PR with fixes after current PR is merged.

Reply:
Fixed in this PR :smile: